public/js/eventsPage.js

```diff
@@ -180,4 +185,15 @@ function validateForm() {
     //Adds all error messages to the messages div.
     document.getElementById('messages').innerHTML = message
     return success
-}
\ No newline at end of file
+}
+
+
+document.querySelector('#submit-search-event').addEventListener('click', function(e) {
+    const searchInput = document.querySelector('#academy-events-search').value
+    e.preventDefault()
+    if ((searchInput.length) && searchInput.length < 255) {
```

Comment:
Check '255'. It should probably be 256, just like in this line:
`if (strlen($eventSearchInput) < 256) {`

Reply:
Updated to 256